Integrations/src/test/java/io/deephaven/integrations/learn/GathererTest.java

```diff
@@ -0,0 +1,113 @@
+package io.deephaven.integrations.learn;
```

Comment:
When you are developing these tests, make sure you run them in IJ with code coverage.  It will let you see if there are any parts of your code that are not tested well.

Reply:
I've run the tests with code coverage, but I don't really know how to interpret the results.  I'll spend some more time on it tomorrow as well